sections/03_categorize.md

```diff
@@ -636,25 +636,24 @@ learned by the model and whether or not a model is sufficiently interpretable to
 identify biases. We discuss the challenge of model interpretability more
```

Comment:
Just spotted "biases". Just before the section in review but my language antenna says "bias" is correct.